openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -160,7 +160,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(37, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
+        with self.assertNumQueries(38, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
```

Comment:
I believe that it is the query to find the user preference for the site theme preview. I'm wondering whether user preferences should all load in one query and then be made available as a cache for subsequent lookups. Currently it appears that each preference lookup results in a separate query. As you said, the additional query didn't seem worth worrying about for now.